src/modules.js

```diff
@@ -0,0 +1,3 @@
+export const todoList = [];
+
+export const todoListElement = document.querySelector('.todo-list');
```

Comment:
Not Enough Modularity, There should be separate Files for backend Functionality and the Code that updates DOM.